packages/next/src/generators/init/utils/eslint.ts

```diff
@@ -4,47 +4,42 @@ import { Linter } from 'eslint';
 
 import { ESLINT_PLUGIN_TESTING_LIBRARY_VERSION } from './constants';
 
-function stacksEslintConfig(projectRootPath: string): Linter.Config {
-    return {
-        extends: [
-            'plugin:@nrwl/nx/react-typescript',
-            'plugin:testing-library/react',
-            'plugin:@next/next/recommended',
-            'next/core-web-vitals',
-        ],
-        ignorePatterns: ['!**/*'],
-        overrides: [
-            {
-                files: ['*.ts', '*.tsx', '*.js', '*.jsx'],
-                parserOptions: {
-                    project: [`${projectRootPath}/tsconfig(.*)?.json`],
-                },
-                rules: {
-                    '@typescript-eslint/no-floating-promises': 'error',
-                    'testing-library/await-async-utils': 'error',
-                    'testing-library/await-async-query': 'error',
-                    'testing-library/no-wait-for-side-effects': 'error',
-                    'testing-library/no-manual-cleanup': 'error',
-                    'testing-library/prefer-explicit-assert': 'warn',
-                    'testing-library/prefer-presence-queries': 'warn',
-                    'testing-library/prefer-wait-for': 'error',
-                    'testing-library/prefer-user-event': 'warn',
-                    'testing-library/no-debug': 'off',
-                },
+const stacksEslintConfig: Linter.Config = {
+    extends: [
+        'plugin:@nrwl/nx/react-typescript',
+        'plugin:testing-library/react',
+        'plugin:@next/next/recommended',
+        'next/core-web-vitals',
+    ],
+    ignorePatterns: ['!**/*'],
+    overrides: [
+        {
+            files: ['*.ts', '*.tsx', '*.js', '*.jsx'],
+            parserOptions: {
+                project: ['./tsconfig(.*)?.json'],
```

Comment:
I dont think this is correct. Eslint runs from the root of the app, so it's going to pick something in the root.